python/GafferUI/SpreadsheetUI.py

```diff
@@ -684,6 +745,27 @@ def __headerButtonPress( self, header, event ) :
 			}
 		)
 
+		sectionNames = _ColumnTabBar.sectionNames( self._qtWidget().model().rowsPlug() )
+		for sectionName in sectionNames :
+			menuDefinition.append(
+				"/Move to Section/{}".format( sectionName ),
+				{
+					"command" : functools.partial( _ColumnTabBar.assignSection, cellPlug, sectionName = sectionName ),
+					"active" : not Gaffer.MetadataAlgo.readOnly( cellPlug ),
```

Comment:
It'd be good to grey out the section that the column is already in.

Reply:
Done in d35774d. As discussed separately, I've rebased everything down ready for a merge.